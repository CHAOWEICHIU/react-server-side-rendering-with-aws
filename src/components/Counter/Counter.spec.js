import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import { StyledBtn } from './'
import 'jest-styled-components'

it('<StyledBtn />', ()=>{
  const tree = renderer.create(
    <StyledBtn />
  ).toJSON()
  expect(tree).toMatchStyledComponentsSnapshot()
})
