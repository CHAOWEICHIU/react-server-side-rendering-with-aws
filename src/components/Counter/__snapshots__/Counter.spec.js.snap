// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledBtn /> 1`] = `


<button
  className="bcQrar"
/>
`;
